packages/avatars/demo/statusindicator.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { StatusIndicator } from '@zendeskgarden/react-avatars';
+
+import { StatusIndicatorStory } from './stories/StatusIndicatorStory';
+
+<Meta title="Packages/Avatars/StatusIndicator" component={StatusIndicator} />
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story
+    name="StatusIndicator"
+    argTypes={{
+      caption: { name: 'caption', control: 'text', table: { category: 'Story' } },
+      show: {
+        name: 'show',
+        control: 'radio',
+        options: ['default caption', 'set caption', 'no caption'],
+        table: { category: 'Story' }
+      }
+    }}
+    args={{
+      show: 'default caption'
+    }}
```

Comment:
Following the convention of other stories:
- the `story` controls can be removed in favor of exposing the `children` prop. Compare with [Tag](https://zendeskgarden.github.io/react-components/?path=/story/packages-tags-tag--tag), [Button](https://zendeskgarden.github.io/react-components/?path=/story/packages-buttons-button--button), and a number of others
- add the `aria-label` override prop. Compare with [IconButton](https://zendeskgarden.github.io/react-components/?path=/story/packages-buttons-iconbutton--icon-button)